custom_components/easee/entity.py

```diff
@@ -190,6 +192,11 @@ def get_value_from_key(self, key):
 
     async def async_update(self):
         """Get the latest data and update the state."""
+#         self._translations = await async_get_translations(
```

Comment:
Is it finished?

Reply:
Removed commented code.
No, translation only is done in frontend. The internal representation of state should not be affected. Can be verified in Developer Tools.